### ISSUE TITLE
cell_methods with standard_names

### ISSUE THREAD

**Issue by zklaus** [IS-ENES-Data/QA-DKRZ#15]
We get a number of annotations of the form
```
annotation: Attribute <msftyz>:cell_methods: Invalid name <grid_longitude:>.
```
the corresponding extract from the output of `ncdump` is
```
        float msftyz(time, basin, lev, rlat) ;
                msftyz:cell_methods = "time: mean grid_longitude: mean" ;
```
this makes use of a feature described in [CF-1.7 Section 7.3.4](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.7/cf-conventions.html#cell-methods-no-coordinates) whereby names in `cell_method` can refer to CF `standard_name`s instead of dimensions or variables in the netcdf file.

This use is demanded by the CMIP6 tables (eg [(Omon, msftyz)](https://github.com/PCMDI/cmip6-cmor-tables/blob/bc7d1c246ec3a3fc2ec9ed59ccf929bb670a0176/Tables/CMIP6_Omon.json#L3131)) and should not be considered an error.

What do you think, @h-dh ?

**Comment by h-dh** [comment]
I totally agree. Unfortunately, there was a missing tiny '!' in the corresponding C++ coding.
Fixed.
Cheers,
hdh

**Comment by zklaus** [comment]
Great, thanks!

PS: If you include `fixes #15` (or any of [a number of alternatives](https://help.github.com/en/articles/closing-issues-using-keywords)) anywhere in the commit message, github will close the issue automatically for you and we get a link to the relevant commit here.
